modules/creating-instance-aws-load-balancer-controller.adoc

```diff
@@ -139,7 +139,7 @@ spec:
 +
 [source,terminal]
 ----
-$ HOST=$(kubectl get ingress -n echoserver echoserver -o json | jq -r '.status.loadBalancer.ingress[0].hostname')
+$ HOST=$(oc get ingress -n echoserver echoserver -o json | jq -r '.status.loadBalancer.ingress[0].hostname')
```

Comment:
@alebedev87 Should we avoid `jq` here?

Reply:
@xenolinux : the below command can be used as replacement of `jq` tool:
```
HOST=$(oc get ingress -n echoserver echoserver --template='{{index .status.loadBalancer.ingress 0}}{{.hostname}}')
```

Reply:
Thanks, @alebedev87 

@dpateriya Can you please update the above command?

Reply:
@dpateriya As per our discussion; I will open a PR for additional changes.